bin/warmup_stats

```diff
@@ -140,16 +145,19 @@ def create_arg_parser():
                         help='Debug level used by logger. Must be one of: '
                              'DEBUG, INFO, WARN, DEBUG, CRITICAL, ERROR')
     parser.add_argument('--language', '-l', dest='language', action='store',
-                        type=str, help='Language under test (in lower-case).')
+                        type=str, help=('Language under test (in lower-case). '
+                                        'Only needed if input files are in CSV format.'))
```

Comment:
Do we need to say somewhere that when using CSV files, only one VM and language combo can be present?

Reply:
Good idea